src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -191,14 +198,14 @@ using LayerTypes = boost::variant<
     NegativeLogLikelihood<arma::mat, arma::mat>*,
     PReLU<arma::mat, arma::mat>*,
     Recurrent<arma::mat, arma::mat>*,
-    RecurrentAttention<arma::mat, arma::mat>*,
+    // RecurrentAttention<arma::mat, arma::mat>*,
```

Comment:
Do you mind to add a ``TODO`` here that explains why we commented the two layer?

Reply:
I have added a comment.